spec/form_models/jobseekers/subscription_form_spec.rb

```diff
@@ -44,12 +44,34 @@
       end
     end
 
-    context "when no criteria are selected" do
-      let(:keyword) { nil }
+    context "when default variant has been applied" do
+      let(:params) { { variant: :default } }
```

Comment:
Can you please test that it does not add an error "when location and no other field are selected"? Which is the difference between the variants.